raidsplit/bot_commands.py

```diff
@@ -30,24 +32,30 @@ async def raid(ctx, raid_name, raiders: commands.Greedy[Member]):
 
     raid_name = raid_name.strip()
     if LocalFile().find_raid(raid_name) is not None:
-        await ctx.send(f"Raid **{raid_name}** already exists please use a different name!")
+        await ctx.send(
+            f"Raid **{raid_name}** already exists please use a different name!"
+        )
         return
 
-    raid = Raid(name=raid_name,
-                admin=UserId(ctx.author.id),
-                proposals={},
-                member_infos={},
-                is_open=True)
+    raid = Raid(
+        name=raid_name,
+        admin=UserId(ctx.author.id),
+        proposals={},
+        member_infos={},
+        is_open=True,
+    )
 
-    await ctx.send(f"A raid wit the name **{raid_name}** and **{len(raiders)}** members was created.")
+    await ctx.send(
+        f"A raid with the name **{raid_name}** and **{len(raiders)}** members was created."
+    )
 
     for raider in raiders:
         new_member: UserId = UserId(raider.id)
         raid.add_member(raider)
         await raider.send(
             f"Hi {raid.member_infos[new_member].name}. Your input has been requested for the **{raid_name}** raid.\n"
-            "Please use the `!split allocate <raid_name> <member_handle> <allocation>`. "
-            "Your allocations should be a number greater 0 and smaller 100 \n\n"
+            f"Please use the `!split allocate {raid_name} <allocations>`. "
+            "Your allocations should be a number greater 0 and smaller 100, and in the same order as the `!split members` command \n\n"
             "If you make a mistake use the `!split allocate` again to modify your allocation."
```

Comment:
```suggestion
            "Your allocations must be percentages between 0 and 100." 
            f"Together, they must sum up to exactly 100 and be specified in the same order as displayed in `!split members {raid_name}`. \n\n"
            f"If you make a mistake use the `!split allocate {raid_name} <allocations>` command again to modify your allocation."
```

Reply:
Minor improvement to stick with the convenient convention of including the current `raid_name` in suggested commands.